README.md

```diff
@@ -1559,102 +2620,168 @@ The route53 types set up various types of Route53 records:
 All Route53 record types use the same parameters:
 
 #####`zone`
-*Required* The zone associated with this record.
+
+Required.
+
+The zone associated with this record.
 
 #####`name`
-*Required* The name of DNS record.
+
+Required.
+
+The name of DNS record.
 
 #####`ttl`
-*Optional* The time to live for the record. Accepts an integer.
+
+Optional.
+
+The time to live for the record. 
+
+Accepts an integer.
 
 #####`values`
-*Required when not using alias_target* The values of the record. Accepts an array.
-*Conflicts with alias_target*
+
+Required.
+
+When not using alias_target* The values of the record. 
```

Comment:
There's a spare * after alias_target, also should have a period after alias target. And I guess we can go nuts and put backticks around alias_target, since it's a setting.